app/results.py

```diff
@@ -218,7 +221,7 @@ def get(self):
             else:
                 # Look for prefix matches.
                 results = self.search(query_dict)
-                results_url = self.get_results_url(self.params.query)
+                results_url = self.get_results_url(self.params.query_name, self.params.query_location)
```

Comment:
Over 80 chars.

Hint: If you use Sublime, you can show a vertical bar at 80th column by following:
http://stackoverflow.com/a/25901060/4339963
(You probably don't want to auto-wrap, but just show the ruler.)